src/commands/deployment/dashboard.ts

```diff
@@ -33,17 +38,51 @@ export const dashboardCommandDecorator = (command: commander.Command): void => {
         );
         return;
       }
-      if (await launchDashboard(opts.port)) {
+      if (await launchDashboard(opts.port, opts.removeAll)) {
         await open("http://localhost:" + opts.port);
       }
     });
 };
 
-export const launchDashboard = async (port: number): Promise<string> => {
+/**
+ * Cleans previously launched spk dashboard docker containers
+ */
+export const cleanDashboarContainers = async () => {
+  let dockerOutput = await exec("docker", [
+    "ps",
+    "-a",
+    "-q",
+    "--filter",
+    "ancestor=samiyaakhtar/spektate:prod",
```

Comment:
one more thing, can we take this `samiyaakhtar/spektate:prod`  from config? in case we change the name of the docker image in future, it's being referenced from `config.introspection!.dashboard!.image!`

Reply:
good point!